assets/terraform-modules/platforms/tinkerbell/workerpool/main.tf

```diff
@@ -21,6 +20,14 @@ storage:
     contents:
       inline: |
         ${var.cluster_name}-worker-${var.name}-${count.index}
+  - path: /etc/systemd/resolved.conf.d/dns_servers.conf
```

Comment:
Can you please add a comment, that this is needed due to limitations of Tinkerbell, cause by https://github.com/tinkerbell/tink/issues/439.